### DISCUSSION TITLE
Displaying SharePoint List item dynamically

### DISCUSSION THREAD

**Discussion by ArturDj10** [microsoft-search/pnp-modern-search#1802] [Q&A]
Hi,

I have just started using modern search and i am not entirely sure if this is possible, but i have a SharePoint list that i want to display on a subsite.

I've been able to reference the list using: 
{searchTerms} Path:https://client.sharepoint.com/sites/SiteName/Lists/listnamecontentclass:STS_ListItem which pulls through all the list items. I am then looking for a way to use the subsite created name to filter the results displayed from the search as i have a column in the list called combined name which would be identical to the subsite name, which would hopefully only display that result.

Every time an item is created in this list a flow will run that will create a subsite from the template, which contains search the search filter and rather than having to manually go in to change the search i was hoping for something dynamic. 

We had previously used Custom query editor and thought we could use Modern search could replace this process, but i cant sadly see how.

Thanks,
Arts

**Comment by wobba** [comment]
If you follow https://microsoft-search.github.io/pnp-modern-search/usage/search-results/tokens/#site-web-hub-etc-tokens then using the Debug layout see if there is a token you can use which reflects the subsite name to use. If you use `{Web.absoluteUrl}` you get the path of the subweb you are at.
